src/main/java/com/zufar/onlinestore/cart/repository/ShoppingSessionItemRepository.java

```diff
@@ -15,4 +15,6 @@ public interface ShoppingSessionItemRepository extends JpaRepository<ShoppingSes
             nativeQuery = true)
     ShoppingSessionItem updateProductsQuantityInShoppingSessionItem(@Param("shopping_session_item_id") UUID shoppingSessionItemId,
                                                                     @Param("products_quantity_change") Integer productsQuantityChange);
+    @Query(value = "DELETE FROM shopping_session_item WHERE id = :shopping_session_item_id RETURNING *", nativeQuery = true)
```

Comment:
1/ Add a space above
2/ Check whether we need to add @Modifying annotation when we use sql delete statement, please

Reply:
fixed